src/main/java/fox/spiteful/forbidden/FMEventHandler.java

```diff
@@ -527,8 +527,9 @@ public void onFeelPain(LivingHurtEvent event) {
                         ent.motionZ += (player.worldObj.rand.nextFloat() - player.worldObj.rand.nextFloat()) * 0.1F;
                     }
                 }
-                for (int x = 0; x < doses; x++) {
-                    ((ItemSubCollar) ForbiddenItems.subCollar).addVis(amulet, primals[randy.nextInt(6)], 1, true);
+                ItemSubCollar collar = ((ItemSubCollar) ForbiddenItems.subCollar);
+                for (int x = 0; x < Math.min(doses, event.entityLiving.getHealth()); x++) {
```

Comment:
`int doses = 3 * (int) event.ammount`, so this should cap out at 3 times the amount of health
```suggestion
                for (int x = 0; x < Math.min(doses, 3 * event.entityLiving.getHealth()); x++) {
```

Reply:
now that you mention it, the cap should also be increased by 3 when doses gets increased by 3. I'll get on fixing that.